.vsts-ci.yml

```
@@ -23,7 +23,8 @@ extends:
       binskim:
         enabled: true
         scanOutputDirectoryOnly: true
-
+        analyzeTargetGlob: +:f|$(Build.SourcesDirectory)\bin\Release\*.exe;+:f|$(Build.SourcesDirectory)\bin\Release\*.dll;-:f|$(Build.SourcesDirectory)\bin\Release\*.test.*
```

Is `analyzeTargetGlob` defined as an "array or string"? If so, this is better:

```yaml
analyuzeTargetGlob:
- +f||...
- +:f|...
```

If not - if it's only a string - you can chomp new lines:

```yaml
analyzeTargetGlob: >-
  +f|...;
  +f|...;
```

This way, it's easier to read each line (notice the `;` is still after each line above) and future diffs are easier to read since each "clause" is on a separate line e.g.,

```suggestion
        analyzeTargetGlob: >-
            +:f|$(Build.SourcesDirectory)\bin\Release\*.exe;
            +:f|$(Build.SourcesDirectory)\bin\Release\*.dll;
            -:f|$(Build.SourcesDirectory)\bin\Release\*.test.*;
```

---

I believe it's parsed as string, I will try the suggested change for better readability!

---

From I read online, the >- gets rid of new lines but there's white space in between them when they are parsed. I couldn't find a way to explicitly parse without any white space. Do you know a way to accomplish this?

---

Shoot. I don't believe you can. Never mind, then.